WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -554,6 +554,7 @@ class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelega
 extension GutenbergViewController {
     private func setupGutenbergView() {
         view.backgroundColor = .white
+        view.tintColor = UIColor.muriel(color: MurielColor(name: .blue))
```

Comment:
In an effort to address the _Editor UI_ references I called out, I researched React Native's `selectionColor` property for `TextInput` a bit. Unfortunately, I was unable to identify where the [default value](https://github.com/facebook/react-native/search?q=selectionColor) for [`selectionColor`](https://reactnative.dev/docs/textinput#selectioncolor) is sourced or why this `tintColor` does not influence `TextInput`.